packages/data-service/src/data-service.ts

```diff
@@ -2549,8 +2552,8 @@ export class DataServiceImpl extends EventEmitter implements DataService {
   }
 
   async createDataKey(
-    provider: ClientEncryptionDataKeyProvider,
-    options?: ClientEncryptionCreateDataKeyProviderOptions
+    provider: any,
+    options?: any
```

Comment:
Let’s maybe still leave comments here about what the types are intended to be?

```suggestion
    provider: any /* ClientEncryptionDataKeyProvider */,
    options?: any /* ClientEncryptionCreateDataKeyProviderOptions */
```